src/MediaVorus/Media/Image.php

```diff
@@ -409,7 +409,11 @@ public function getColorSpace()
      */
     protected function extractFromDimensions($WidthXHeight)
     {
-        $values = explode('x', strtolower($WidthXHeight));
+        if (strpos($WidthXHeight, 'x') !== false) {
+            $values = explode('x', strtolower($WidthXHeight));
+        } else {
+            $values = explode(' ', strtolower($WidthXHeight));
+        } 
```

Comment:
better to use a regex, so futur changes have a chance not to break again : [https://regex101.com/r/UrZGJD/2](https://regex101.com/r/UrZGJD/2)

Reply:
ok